internal/chartverifier/checks/checks_test.go

```diff
@@ -549,51 +549,6 @@ func TestRequiredAnnotationsPresent(t *testing.T) {
 	}
 }
 
-func TestSemVers(t *testing.T) {
```

Comment:
Any particular reason this test needs to change? Has this test been moved to the replacing library?

Reply:
These are the unit tests for the `getOCPRange` function, which is removed from this repo with this commit.

The Go library implements unit tests too, see https://github.com/opdev/getocprange/blob/main/get-ocp-range_test.go